cupy/cuda/compiler.py

```diff
@@ -244,13 +251,15 @@ def compile_using_nvcc(source, options=(), arch=None,
             assert False, code_type
 
 
-def _preprocess(source, options, arch, backend):
+def _preprocess(source, options, arch, backend, log_stream):
```

Comment:
I am sorry, Matt, this is my bad. I looked again and no longer think we need `log_stream` in this `_preprocess()` function. The purpose of this function is to check the compiler config by compiling an empty source code.

Reply:
By not touching this function, it also ensures we don't write to the same file twice (although it has no harm).